x-pack/plugins/uptime/server/rest_api/pings/journey_screenshots.ts

```diff
@@ -63,22 +50,11 @@ export const createJourneyScreenshotRoute: UMRestApiRouteFactory = (libs: UMServ
         },
       });
     } else if (isRefResult(result)) {
-      const blockIds = result.screenshot_ref.blocks.map(({ hash }) => hash);
-      let blocks: ScreenshotBlockDoc[];
-      try {
-        blocks = await libs.requests.getJourneyScreenshotBlocks({
-          uptimeEsClient,
-          blockIds,
-        });
-      } catch (e: unknown) {
-        return response.custom({ statusCode: 500, body: { message: e } });
-      }
       return response.ok({
         body: {
           screenshotRef: result,
-          blocks,
         },
-        headers: getSharedHeaders(result.synthetics.step.name, result.totalSteps ?? 0),
+        headers: getSharedHeaders(result.synthetics.step.name, result.totalSteps),
```

Comment:
Why did we remove the fallback?

Reply:
`totalSteps` [is required](https://github.com/elastic/kibana/blob/master/x-pack/plugins/uptime/server/lib/requests/get_journey_screenshot.ts#L17), so we didn't need the fallback to begin with. Plus, it comes from `hits.total.value`, so it should pretty much always be available.